### ISSUE TITLE
Allow changing the watched command at runtime with a keybinding

### ISSUE THREAD

**Issue by fritzrehde** [fritzrehde/watchbind#58]
This would make watchbind a lot more powerful, and would be required for implementing a little TUI file manager (if we go into a directory, we'd have to change what we are `ls`ing).

**Comment by fritzrehde** [comment]
I don't think this is required for building the TUI file manager, actually. Yes, the input command needs to change somehow, but the structure of the input command stays the same. What changes is some sort of state, in the case of the file manager that would be some sort of state. v0.1.18 introduces such state through environment variables, so building a file manager is now quite trivial. An example for how to do this will be added to the README in the future.
TLDR: I thought this feature would be needed for something I want to do, but it turns out it isn't. For now, I don't see any use-case where changing the whole command would be required, so I won't prioritize implementing this unless someone provides me with a valid use-case.
